sgl-kernel/csrc/cpu/gemm.cpp

```diff
@@ -487,7 +487,7 @@ at::Tensor weight_packed_linear(at::Tensor& mat1, at::Tensor& mat2, std::optiona
   const float* bias_data = nullptr;
   if (has_bias) {
     CHECK_EQ(bias.value().size(0), N);
-    bias_data = bias.value().data_ptr<float>();
+    bias_data = bias.value().to(at::kFloat).data_ptr<float>();
```

Comment:
we don't need `to.(at::kFloat)` here. our cpu_opt_ww11 branch doesn't have this.

if the model has non-float32 bias, we need to convert when packing the weights to vnni formats in python APIs not the C++ ones.

Reply:
It is introduced by https://github.com/mingfeima/sglang/commit/6f73059a93758b49f428b5739f188e6bb9bd27cd, I remove this line change.